### ISSUE TITLE
Problems with uploading non-image attachments

### ISSUE THREAD

**Issue by moorglade** [helpyio/helpy#427]
When creating a new discussion, if I attach a non-image (eg. plain text) file, the resulting discussion page's content is not displayed correctly (only the page's header, title and footer are shown).

If I try to attach such a file to a comment in an existing discussion, a generic error is shown and the comment is not added.

In both cases, the problematic files are not saved to the `/public/uploads` directory.

The system also does not suggest in any way that uploading non-image files is not permitted.

**Helpy version**: 1.0
**Upload backend**: file system

**Comment by scott** [comment]
Are you using Heroku?

**Comment by scott** [comment]
Also, could you please clarify if you are seeing this on the front-end community, or only in the admin?

**Comment by moorglade** [comment]
No, we're not using Heroku, just building from source on our private server.

I'm seeing problems in both the admin and the community views. I attached some screenshots to illustrate the issue.

Creating a new discussion with an attachment
![helpy-1](https://cloud.githubusercontent.com/assets/8457512/19923859/a8c94af8-a0e9-11e6-8512-19e0fa586613.png)

Community view - discussion "body" not displayed
![helpy-2](https://cloud.githubusercontent.com/assets/8457512/19923858/a8c6c4fe-a0e9-11e6-9642-fb442649a4d6.png)

Admin view - message content and attachment not displayed
![helpy-3](https://cloud.githubusercontent.com/assets/8457512/19923857/a8c54e12-a0e9-11e6-8cc3-05a3158f2d80.png)

**Comment by scott** [comment]
From your screenshots, it looks like the file does not have an extension of `txt`  This could be causing the problem.  I tested it against a `file.txt` and it worked correctly, however when I removed the filetype I was able to get the above behavior.

Either way this is not idea, we need better handling of files that are not accepted or identifiable.
